extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -14,9 +14,13 @@
  * limitations under the License.
  */
 
+import * as st from '../../../src/style';
+import * as tr from '../../../src/transition';
```

Comment:
Ditto

Reply:
Would the following:
```
import {setStyles} as st from '../../../src/style';
import {px, numeric, setStyles} as tr from '../../../src/transition';
```

not allow you to say
`st.setStyles` and `tr.setStyles`?

Reply:
Nope, but it turns out that you can do: 
```
import { export1 , export2 as alias2 , [...] } from "module-name";
```

per [MDN import docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)

Reply:
Ohhh can we do that instead? I am a bit wary about importing things that aren't used in this file.

It's optional though, feel free to land without this change.

Reply:
I see you have already done that. LGTM :)